cassandra/cqlengine/columns.py

```diff
@@ -72,6 +73,7 @@ def get_property(self):
             return property(_get, _set)
 
 
+@total_ordering
```

Comment:
I could add this backport https://pypi.python.org/pypi/total-ordering

Or just add all the ordering methods to the class. Whichever you prefer